modules/gitops-argo-cd-properties.adoc

```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
-  * _<Replicas>_ - The number of replicas for the Argo CD server. Must be greater than or equal to `0`. If `Autoscale` is enabled, `Replicas` is ignored.
-  * _<Route>_ - Route configuration options.
-  * _<Service.Type>_ - The `ServiceType` used for the service resource.
-  * _<LogLevel>_ - The log level to be used by the Argo CD Server component. Valid options are  `debug`, `info`, `error`, and `warn`.
-  * _<LogFormat>_ - The log format used by the Argo CD Application Controller component. Valid options are `text` or `json`.
-  * _<Env>_ - Environment to set for the server workloads.
-|`SSO` |Single Sign-on options.|`__<Object>__`
-a|* _<Image>_ - The container image for Keycloak. This overrides the `ARGOCD_KEYCLOAK_IMAGE` environment variable.
-  * _<Keycloak>_ - Configuration options for Keycloak SSO provider.
-  * _<Dex>_ - Configuration options for Dex SSO provider.
-  * _<Provider>_ - The name of the provider used to configure Single Sign-on. For now the supported options are Dex and Keycloak.
-  * _<Resources>_ - The container compute resources.
-  * _<VerifyTLS>_ - Whether to enforce strict TLS checking when communicating with Keycloak service.
-  * _<Version>_ - The tag to use with the Keycloak container image.
-|`StatusBadgeEnabled` |Enable application status badge.|`true` |
-|`TLS` |TLS configuration options.|`__<Object>__`
-a|* _<CA.ConfigMapName>_ - The name of the `ConfigMap` which contains the CA certificate.
-  * _<CA.SecretName>_ - The name of the secret which contains the CA Certificate and Key.
-  * _<InitialCerts>_ - Initial set of certificates in the `argocd-tls-certs-cm` config map for connecting Git repositories via HTTPS.
-|`UserAnonyousEnabled` |Enable anonymous user access.|`true` |
-|`Version` |The tag to use with the container image for all Argo CD components.|Latest Argo CD version|
-|`Banner` |Add a UI banner message.|`__<Object>__`
-a|* _<Banner.Content>_ - The banner message content (required if a banner is displayed).
-  * _<Banner.URL.SecretName>_ - The banner message link URL (optional). 
+|`applicationInstanceLabelKey` |The `metadata.label` key name where Argo CD injects the app name as a tracking label.|`app.kubernetes.io/instance` |
+|`applicationSet` 
+|`applicationSet` controller configuration options.
+| `_<object>_`
+a|* _<image>_ - The container image for the `ApplicationSet` controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
+  * _<version>_ - The tag to use with the `ApplicationSet` container image.
+  * _<resources>_ - The container compute resources.
+  * _<logLevel>_ - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
+  * _<logFormat>_ - The log format used by the Argo CD Application Controller component. Valid options are `text` or `json`.
+  * _<parallelismLimit>_ - The kubectl parallelism limit to set for the controller `(--kubectl-parallelism-limit flag)`.
```

Comment:
Just checking, ignore if no change is needed - are the parentheses something that you would include when using this flag, or is it used grammatically to introduce extra context to the description? Point being, if the parentheses are only used here in the description and not in the command, I wouldn't include it in the monospace styling:
```suggestion
  * _<parallelismLimit>_ - The kubectl parallelism limit to set for the controller (`--kubectl-parallelism-limit` flag).
```